packages/debug-utils/package.json

```diff
@@ -23,7 +23,6 @@
     "bn.js": "^5.1.3",
     "chalk": "^2.4.2",
     "debug": "^4.3.1",
-    "highlight.js": "^10.4.0",
```

Comment:
Maybe make sure that chromafi doesn't need this as a peerDependency?

Reply:
`highlight.js` is a [dependency of chromafi](https://github.com/F1LT3R/chromafi/blob/master/package.json#L29)